backend/de.metas.business/src/main/java/de/metas/invoice/InvoiceCreditContext.java

```diff
@@ -43,4 +44,9 @@
 	 *         the credited quantities to be invoiced once more. Note that if the invoice's grand total is already partially allocated, this value will be override with <code>false</code>.
 	 */
 	boolean creditedInvoiceReinvoicable;
+
+	/**
+	 * @return if <code>true</code>, this means that this invoice should not be changeable after creation. In this case the invoice should also be completed.
+	 */
+	boolean fixedInvoice;
```

Comment:
a completed invoice cannot be changed anyways. You mean if that flag is true than a completed credit memo is generated and that credit memo cannot be reversed/void/reactived afterwards. right?

Reply:
yes correctly